wagn/features/step_definitions/email_steps.rb

```diff
@@ -95,11 +95,15 @@ def current_email_address
   expect(current_email).to have_subject(Regexp.new(text))
 end
 
-Then /^(?:I|they) should see [\|"](.*)[\|"] in the email body$/ do |text|
+Then /^(?:I|they) should see \|(.*)\| in the email body$/ do |text|
   expect(current_email.text_part.body.raw_source).to include("#{text}")
 end
 
-Then /^(?:I|they) should see \/([^"]*?)\/ in the email body$/ do |text|
+Then /^(?:I|they) should see "(.*)" in the email body$/ do |text|
+  expect(current_email.text_part.body.raw_source).to include("#{text}")
+end
+
```

Comment:
Don't know if we need or want them, just making them work.  At first I thought I'd use the '/' form, but that delimits a regexp.  The other case allowed double quotes or pipes, but I didn't trust the combined regexp so I made two.